traefik/tests/metrics-config_test.yaml

```diff
@@ -5,28 +5,6 @@ capabilities:
   majorVersion: 1
   minorVersion: 16
 tests:
-  - it: should set default values for args
```

Comment:
While I agree it has nothing to do here, I think it's still relevant to cover this.
Please have a look to my commit :-)